tools/check-nfkc.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+
+# Checks if all normative and informative
+# WGSL spec sources are NFKC normal
+# Note: Filters sources by the working directory,
+# any other directory can benefit as well as WGSL
+
+import unicodedata
+import subprocess
+import sys
+
+files = subprocess.run(
+    ["git", "ls-files", "--recurse-submodules"], check=True, capture_output=True
```

Comment:
The "--recurse-submodules" is probably more than we need.

Reply:
JIC if the script had wider use in repo, removed in new commit